sdk/test-utils/recorder/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 
-## 3.1.0 (2023-03-14)
+## 3.1.1 (2024-04-09)
+
+### Bugs Fixed
+
+- Minor bug where min version testing in min-max testing fails because the types of `@azure/core-client` is not extensive. Bumping the `@azure/core-client` version to latest. [#27187](https://github.com/Azure/azure-sdk-for-js/pull/27187)
```

Comment:
```suggestion
- Minor bug where min version testing in min-max testing fails because the types of `@azure/core-client@1.0.0` is not extensive. Bumping the `@azure/core-client` version to latest. [#27187](https://github.com/Azure/azure-sdk-for-js/pull/27187)
```